sql/patch.sql

```diff
@@ -88,3 +88,6 @@ UPDATE `list_options`
 SET `notes` = '{"form_title":{"presence": {"message": "Title Required"}}}'
 WHERE `list_id` = 'page_validation' AND `option_id` = 'add_edit_event#theform_prov' AND `title` = '/interface/main/calendar/add_edit_event.php?prov=true' AND `notes` = '{}';
 
+#IfTable track_events
+ALTER TABLE `track_events` DROP INDEX `unique_event_label`, ADD UNIQUE `unique_event_label` (`event_label`, `event_url`(255));
```

Comment:
recommend renaming the index on change and instead checking the index via #IfNotIndex
(this way the index won't keep getting reset when run upgrade script more than once)